src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -190,6 +190,15 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
             'You are not authorized to view this page.');
       });
     });
+
+    // Display note on submission on mobile device
+    if (window.innerWidth < 768 && this.feedbackSessionSubmissionStatus === FeedbackSessionSubmissionStatus.OPEN) {
```

Comment:
I don't think this works... on doing local testing, I am still seeing the "Note On Submission" prompt pop up when viewing the submission of a closed session.

Reply:
I see... I will change it then.